packages/react/src/Suspense.tsx

```diff
@@ -4,42 +4,61 @@ import { useIsClient } from './hooks'
 import { type PropsWithDevMode } from './utility-types'
 import { noop } from './utils'
 
-export interface SuspenseProps extends PropsWithDevMode<SuspenseDevModeOptions>, ReactSuspenseProps {}
+export interface SuspenseProps extends PropsWithDevMode<SuspenseDevModeOptions>, ReactSuspenseProps {
+  clientOnly?: boolean
+}
```

Comment:
```suggestion
export interface SuspenseProps extends PropsWithDevMode<SuspenseDevModeOptions>, ReactSuspenseProps {
  /**
   * `<Suspense/>` will return fallback in server. After mount(in client) return children. Since mount only happens on the client, server-side rendering can be avoided.
   * @see https://suspensive.org/docs/react/Suspense#avoid-server-side-rendering-clientonly
   */
  clientOnly?: boolean
}
```

what about add JSDoc for this line?

### Before

<img width="664" alt="image" src="https://github.com/suspensive/react/assets/57122180/264e3839-4a5f-41c5-9064-ad5fba0e0ad4">

### After

<img width="613" alt="image" src="https://github.com/suspensive/react/assets/57122180/8b708ec4-3d36-4624-aec5-c1612847d60b">

Reply:
So cool!